### PR TITLE
repair-bde: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1579

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/windows/repair-bde.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# repair-bde
+
+> Attempt to repair or decrypt a damaged BitLocker-encrypted volume.
+
+- Attempt to repair a specified volume:
+
+`repair-bde {{input_volume}}`
```
Comment (agnivade) [inline pages/windows/repair-bde.md]: What does the input_volume and output_volume look like ? Can we give an example ?

Comment (pxgamer) [reply pages/windows/repair-bde.md]: Input and output volumes can have 2 formats:
- the drive letter with a colon. (e.g. `C:`)
- the id of the volume (e.g. `\\?\Volume{id}\`)

Comment (agnivade) [reply pages/windows/repair-bde.md]: I think going with a drive letter will make things clearer.

Comment (pxgamer) [reply pages/windows/repair-bde.md]: Sounds good. :+1:

**Pull request merged**